src/main/java/dinkplugin/notifiers/data/CollectionNotificationData.java

```diff
@@ -28,13 +31,32 @@ public class CollectionNotificationData extends NotificationData {
     @Nullable
     Integer totalEntries;
 
+    @Nullable
+    String dropperName;
+
+    @Nullable
+    LootRecordType dropperType;
+
+    @Nullable
+    Integer dropperKillCount;
+
     @Override
     public List<Field> getFields() {
-        if (completedEntries == null || totalEntries == null)
-            return super.getFields();
-
-        return Collections.singletonList(
-            new Field("Completed Entries", Field.formatProgress(completedEntries, totalEntries))
-        );
+        List<Field> fields = new ArrayList<>(2);
+        if (completedEntries != null && totalEntries != null) {
+            fields.add(
+                new Field("Completed Entries", Field.formatProgress(completedEntries, totalEntries))
+            );
+        }
+        if (dropperKillCount != null) {
+            assert dropperName != null && dropperType != null;
+            fields.add(
+                new Field(
+                    Drop.getAction(dropperType) + " Count: " + dropperName,
```

Comment:
is this fine (i.e., `Kill Count: NPC name` as field title), or should this be split into two separate fields (source & kill count)

Reply:
I prefer it as two separate fields, without looking at it

Reply:
Done in https://github.com/pajlads/DinkPlugin/pull/424/commits/e97606c0ad85b59e833bb82919335e1f98d120bb